models/datasets/generic/ds__appointments.sql

```diff
@@ -0,0 +1,39 @@
+select
+    p.id as patient_id,
+    p.display_id,
+    p.first_name,
+    p.last_name,
+    p.date_of_birth,
+    date_part('year', age(a.start_datetime::date, p.date_of_birth::date)) as age,
+    initcap(p.sex::text) as sex,
```

Comment:
`date_of_birth` and `sex` should already be formatted.
```suggestion
    date_part('year', age(a.start_datetime::date, p.date_of_birth)) as age,
    sex,
```